src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.SymbolMoniker.cs

```diff
@@ -28,9 +28,6 @@ internal abstract partial class AbstractFindUsagesService
             if (moniker == null)
                 return;
 
-            // Let the find-refs window know we have outstanding work
-            await using var _ = await context.ProgressTracker.AddSingleItemAsync().ConfigureAwait(false);
```

Comment:
intentional removal.  progress tracking doesn't work well for richnav as we don't really know how many items it will need to search.  so trying to evne add progress items for it just ends up interfering with the good progress bar we have for the normal find-refs for user code in their solution.

Reply:
Ah, so we did all that refactoring work to then not use it? :smile:

Reply:
it's used elsewhere :)   just here it's not particularly good and def degrades the experience.  i.e. if you do have code-index, we just jump to the 50% point immediately because it says "i have one piece of work", and then finishes it.  The other part of find refs has has added one piece of owrk that is *not* done, causing us to be at 1-of-2 done.

This is unlike normal find refs, which can add a bunch of known work it needs to do (like looking at all files with bloom filter matches) which gives far more fine-grained progress.